Lib/test/test_argparse.py

```diff
@@ -5224,6 +5224,192 @@ def test_nargs_3_metavar_length3(self):
         self.do_test_no_exception(nargs=3, metavar=("1", "2", "3"))
 
 
+class TestAddArgumentMetavarWrapNoException(TestCase):
+    """Check that certain special character wrap with no exceptions
+    Based off TestAddArgumentMetavar"""
+
```

Comment:
Please combine all of these into a single test method that runs the cases in a loop with unittest subtests.